src/plugins/intel_cpu/tests/functional/subgraph_tests/src/param_result_custom_blob.cpp

```diff
@@ -2,16 +2,17 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "shared_test_classes/subgraph/parameter_result.hpp"
 #include "common_test_utils/test_constants.hpp"
+#include "shared_test_classes/subgraph/parameter_result.hpp"
 
 using namespace SubgraphTestsDefinitions;
+using namespace ov::test;
 using namespace InferenceEngine;
 
 namespace CPULayerTestsDefinitions {
 
 class ParameterResultCustomBlobTest : public ParameterResultSubgraphTestLegacyApi {
```

Comment:
Change inheritance to new API? (or remove, because of `legacy` api)

Reply:
I kept both implementations as is right now